docs/tutorials/tabular_prediction/tabular-deployment.md

```diff
@@ -173,6 +179,26 @@ Optimized:
 predictor_clone_opt.get_size_disk_per_file()
 ```
 
+In order to further improve inference efficiency, we can call `.compile_models()` to automatically
```

Comment:
Lets add a section header here so we can link to it when answering user questions

Reply:
Added.